### PR TITLE
DESCW-534 New Projects CPT Actions + Projects CPT

### PR DESCRIPTION
**Pull request by codewisenate**
+ BCGov Accordion to theme using ES6, patterns updated/added

### CODE REVIEW

**Comment by codewisenate** [comment]
Added Projects CPT, two querys: toggle for Actions and accordion for Projects, added BCGov Accordion to theme, refactored away from jQuery/Bootstrap, now functions with ES6, various styles and patterns updated/added

File: src/custom-types/cleanbc/cleanbc-projects.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?php
```
Comment (ShawnTurple) [inline src/custom-types/cleanbc/cleanbc-projects.php:1]: this was not the intent of this theme to add cpt...

Comment (codewisenate) [reply src/custom-types/cleanbc/cleanbc-projects.php:1]: They can be easily excluded but it leans into the FSE experience.

Comment (ShawnTurple) [reply src/custom-types/cleanbc/cleanbc-projects.php:1]: for now, i'm ok with leaving this, in as we(you) are exploring options.

File: src/scripts/admin/blocks/collapse/container/deprecated.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';
```
Comment (ShawnTurple) [inline src/scripts/admin/blocks/collapse/container/deprecated.js:1]: you know you don't need deprecated, unless you have stuff in production... you probably could get rid of this file.

File: src/scripts/admin/blocks/collapse/container/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks } from '@wordpress/block-editor';
+import { deprecated } from './deprecated';
+import { attributes } from './attributes';
+
+registerBlockType('bcgov/collapse', {
```
Comment (ShawnTurple) [inline src/scripts/admin/blocks/collapse/container/index.js:7]: might want to change the namespace, i'm guessing is the original one..  And keep consistent for all blocks you might create.

File: src/scripts/admin/blocks/collapse/container/public.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+			item.classList.add('hide');
+			setTimeout(function() {
+				item.classList.add('collapse');
+				item.classList.remove('collapsing');
+			}, 300);
+		});
+	}
+}
+
+export function collapseItemsButtonClick() {
```
Comment (ShawnTurple) [inline src/scripts/admin/blocks/collapse/container/public.js:54]: maybe some docblocks.

File: src/scripts/admin/blocks/collapse/item/attributes.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+export const attributes = {
```
Comment (ShawnTurple) [inline src/scripts/admin/blocks/collapse/item/attributes.js:1]: again. just be careful of the selector semantic here.

File: src/scripts/admin/blocks/collapse/item/index.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import { InnerBlocks, RichText } from '@wordpress/block-editor';
+import { attributes } from './attributes';
+
+registerBlockType('bcgov/collapse-item', {
```
Comment (ShawnTurple) [inline src/scripts/admin/blocks/collapse/item/index.js:6]: namespace change?

File: templates/single-cleanbc-projects.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<!-- wp:template-part {"slug":"header","theme":"bcgov-wordpress-block-theme","tagName":"header","lock":{"move":true,"remove":true},"className":"site-header"} /-->
```
Comment (ShawnTurple) [inline templates/single-cleanbc-projects.html:1]: I worry that this is polluting the core of the theme to be site agnostic.... Just observation.

**Review by ShawnTurple — APPROVED** [review]

File: templates/single-cleanbc-projects.html (hunk 7 above)
Comment (codewisenate) [reply templates/single-cleanbc-projects.html:1]: Agreed, I will pull the project specific CPT and we'll either work around it or come up with an alternate solution at a later date.

File: src/scripts/admin/blocks/collapse/item/index.js (hunk 6 above)
Comment (codewisenate) [reply src/scripts/admin/blocks/collapse/item/index.js:6]: Honestly thought I had updated that. Must have seen a squirrel when doing so.

File: src/scripts/admin/blocks/collapse/container/deprecated.js (hunk 2 above)
Comment (codewisenate) [reply src/scripts/admin/blocks/collapse/container/deprecated.js:1]: Good call.

**Pull request merged**